.github/ISSUE_TEMPLATE/minor-release.md

```diff
@@ -0,0 +1,14 @@
+---
+name: VRL minor release
+about: Use this template for a new minor release.
+title: "VRL [version] release"
+labels: "domain: releasing"
+---
+
+- [ ] Create release preparation PR
+  - [ ] Bump [Cargo.toml](https://github.com/vectordotdev/vrl/blob/main/Cargo.toml#L3) version and commit the change.
```

Comment:
I'd suggest adding to run `cargo check` to update `Cargo.lock` too.

Reply:
Good point, added. (My IDE does it automatically so I always skipped this)